lib/src/isolate.dart

```diff
@@ -318,6 +329,35 @@ class VMIsolateRef {
     }
   }
 
+  Future<VMReloadReport> reloadSources(
```

Comment:
Document this. In general, we try to make our documentation at least as good as the protocol documentation, and better where possible.

Also add a note indicating what protocol version this was added in.